### ISSUE TITLE
Connection to remote MSSQL server fails with SSL issue

### ISSUE THREAD

**Issue by shahmanash** [gbif/ipt#2513]
Connection to a remote MSSQL server from an IPT running in docker fails with SSL related error, see attached screenshot.

![ms-sql-server-tls-error](https://github.com/user-attachments/assets/c4f16a09-f051-418d-ad94-959ef30d25a5)

The testing (temporary) fix is to add ``trustServerCertificate=true;`` to the JDBC url definition in the **jdbc.properties file**
i.e. Change 
https://github.com/gbif/ipt/blob/master/src/main/resources/jdbc.properties#L25

`mssql.url=jdbc:sqlserver://{host};databaseName={database}`
to 
`mssql.url=jdbc:sqlserver://{host};databaseName={database};trustServerCertificate=true;`

However it is not persistant in a docker container and probably not the best way in a prod environment.

Please advise on the best possible way to address the issue.

**Comment by mike-podolskiy90** [comment]
Thank you for contacting us. We'll get back to you.

**Comment by bart-v** [comment]
Just add `;encrypt=false` to your server name, so it becomes
`Ring;encrypt=false`

**Comment by shahmanash** [comment]
Thanks, 
I can confirm that the suggested fix of adding the `;trustServerCertificate=true` or `;encrypt=false` to the value in the Database field works.
But it might not be the most elegant way to solve the issue.

**Comment by mike-podolskiy90** [comment]
Yes, it seems it works as a workaround right now. We'll make a fix to address it properly
